lib/components/SLoginPage.vue

```diff
@@ -15,37 +19,68 @@ export interface CoverPhotographer {
   link: string
 }
 
-export interface Action {
+export type Action = ActionPassword | ActionSocial
+
+export interface ActionPassword {
+  type: 'password'
+  onSubmit(email: string, password: string): Promise<void>
+}
+
+export interface ActionSocial {
   type: 'google'
-  onClick: () => Promise<void>
+  onClick(): Promise<void>
```

Comment:
maybe we should make both onSubmit in v4 👀

Reply:
Yeah good point 👀